markdown/types_and_values.md

```diff
@@ -0,0 +1,83 @@
+**#Part One** :  
+**##Identifying the Type of Given Values.**  
+_###Instructions: For each of the following Elm values, write down its most specific type._
+
+1.  42 - Int
+2.  True - Bool
+3.  3.14 - Float
+4.  'a' - Char
+5.  [1, 2, 3] - List Int
+6.  "Hello, Elm!" - String
+7.  ['a', 'b', 'c'] - List Char
+8.  (True, 'a') - (Bool, Char)
+9.  Just "Hello" - Maybe String
+10. Nothing - Maybe a
+11. (4 // 1) - Int
+12. [] - List a
+13. [True, False, True] - List Bool
+14. Just Nothing - Maybe (Maybe a)
+15. [Just 5, Nothing, Just 10] - List (Maybe Int)
+16. [Just 'a', Nothing] - List (Maybe Char)
+17. [(1, 'a'), (2, 'b')] - List (Int, Char)
+18. [(True, "Yes"), (False, "No")] - List (Bool, String)
+19. Just (Just 'a') - Maybe (Maybe Char)
+20. ("Hello", [1, 2, 3]) - (String, List Int)
+21. (42, [True, False], Just 'c') - (Int, List Bool, Maybe Char)
+22. [(True, 42), (False, 24)] - List (Bool, Int)
+23. [Just "Hello", Nothing, Just "World"] - List (Maybe String)
+24. ("Elm", Just [False, True], ('a', 42)) - (String, Maybe (List Bool), (Char, Int))
+25. Just True ('a' "chelz") [] - (Maybe Bool, (Char, String) List a)
+
+**#Part Two** :  
+**##Providing Values That Satisfy Specified Types.**  
+_###Instructions: For each of the following Elm types, provide three values that satisfy the type._
+
+1.  Int: 0, 42, -10
```

Comment:
Remember that these are just `numbers`. To turn them into `Int`s you need to perform integer division (by 1) on them.

For example: `0 // 1`